operator/src/main/java/oracle/kubernetes/operator/helpers/ConfigMapHelper.java

```diff
@@ -380,6 +401,27 @@ public NextAction apply(Packet packet) {
         ScanCache.INSTANCE.registerScan(
             info.getNamespace(), info.getDomainUid(), new Scan(wlsDomainConfig, new DateTime()));
         packet.put(ProcessingConstants.DOMAIN_TOPOLOGY, wlsDomainConfig);
+        if (miiDomainZipHash != null) {
+          packet.put(ProcessingConstants.DOMAIN_HASH, miiDomainZipHash);
+        }
+        if (miiModelSecretsHash != null) {
+          packet.put(ProcessingConstants.SECRETS_HASH, miiModelSecretsHash);
+        }
+        String domainRestartVersion = info.getDomain().getAdminServerSpec().getDomainRestartVersion();
```

Comment:
Don't use getAdminServerSpec().  This could be info.getDomain().getSpec().getRestartVersion();

Reply:
There is no getRestartVersion in the spec.The restartVersion is in the server spec.

Reply:
I checked... you are incorrect ;).  It's in the base class.

Reply:
Okay, I figured out what I'd missed before.  The field `restartVersion` is on base config because you can have different values at the domain level, cluster, or on a specific server.  But `introspectVersion` is only at top-level.  I have a fix for what I'm thinking.

Reply:
Thanks

Reply:
@rjeberhard Please resolve this comment if/when the fix is in place.